enki/exceptions.py

```diff
@@ -26,6 +26,18 @@
 """
 
 
+def check_errors(data):
```

Comment:
I would move this to the ServerTaskRunsLoader class as a method. I believe it is its responsibility to know if a response from the server is ok or not.